### DISCUSSION TITLE
13.0.0

### DISCUSSION THREAD

**Discussion by SubJunk** [UniversalMediaServer/UniversalMediaServer#3704] [General]
This release is all about 3D and virtual reality, thanks to contributions by @threedguru. He has done [a blog post to explain how it works in detail](https://iwantaholodeck.com/on-the-fly-2d-to-3d-video-conversion-with-universal-media-server-ums/).

This is an example of what it can look like in the VR environment:
![BigScreenExample-1](https://user-images.githubusercontent.com/584228/208194669-45e4fce1-19ba-4f6e-ac35-0ff16102f9cd.png)

Here's an overview of what is happening above:
<img width="717" alt="3d-vr-overview" src="https://user-images.githubusercontent.com/584228/208194750-37310485-2ea6-4997-afd3-513abcd359ad.png">

In addition to this very cool feature, there are big improvements to LG TV and VLC support, major improvements to API metadata fetching and network use, and important bugfixes.

### General:
- Added 2D to 3D conversion for virtual reality, for full details see https://iwantaholodeck.com/on-the-fly-2d-to-3d-video-conversion-with-universal-media-server-ums/
- Fixed reload button is disabled after a restart
- Fixed sometimes mixed renderer found
- Implements LINN iOS app search request for attribute upnp:artist@role=composer
- Handle UPNP:AlbumArtist
- Escape iOS Smart Punctuation apostrophe
- Fixed case-insensitive UPnP search
- Double-click on tray icon opens web settings
- Made language more clear on the right-click tray menu
- Fixed API metadata countries, plot, ratings, rated (classification), start year, tagline, total seasons, and votes
- Fixed API poster/cover images sometimes not being set
- Fixed duplicate API fetches for failed results
- Fixed star rating updates not immediately updating
- Fixed resume files with AviSynth transcoding

### Translation updates via Crowdin
- Czech (94%)
- Danish (99%) (thanks, NCAA!)
- Finnish (99%) (thanks, Esko Gardner!)
- French (94%) (thanks, SurfaceS!)
- German (93%) (thanks, jaba82!)
- Korean (94%) (thanks, VenusGirl!)
- Portuguese (100%) (thanks, mariopinto!)
- Portuguese (Brazilian) (94%) (thanks, Sandro Almeida!)
- Spanish (88%) (thanks, edwardalvarez2011!)
- Turkish (100%) (thanks, Burak Yavuz!)
- Ukrainian (26%) (thanks, Paul Furlet!)

### Renderers:
- Let LG TVs use their built-in resume only
- Fixed support for DTS on LG OLED models newer than 2019
- Avoid re-encoding x265 needlessly on LG OLEDs
- Detect more versions of VLC

### Dependencies:
- Updated all Node.js subdependencies
- Update dependency @types/jest to v29.2.4
- Update dependency @types/lodash to v4.14.191
- Update dependency @types/node to v18.11.13
- Update dependency @types/react to v18.0.26
- Update dependency @types/video.js to v7.3.50
- Update dependency axios to v1.2.1
- Update dependency com.github.oshi:oshi-core to v6.4.0
- Update dependency com.puppycrawl.tools:checkstyle to v10.5.0
- Update dependency eslint to v8.29.0
- Update dependency hls.js to v1.2.9
- Update dependency react-router-dom to v6.4.5
- Update dependency tabler-icons-react to v1.56.0
- Update dependency typescript to v4.9.4
- Update Mantine monorepo packages to v5.8.4
- Update typescript-eslint monorepo to v5.46.0

<hr /><em>This discussion was created from the release <a href='https://github.com/UniversalMediaServer/UniversalMediaServer/releases/tag/13.0.0'>13.0.0</a>.</em>

**Comment by TowardAnarres** [comment]
THANKS A MILLION!
Seriously, I can't explain how great this is. 
My family is going to be blown away by Miyazaki movies in 3D. :)

**Reply by threedguru** [reply]
That's really fantastic to hear. It sounds like it's working well for you. That's great feedback.
